integration-tests/src/test/java/oracle/kubernetes/operator/ItOperatorUpgrade.java

```diff
@@ -89,190 +106,82 @@ public void cleanupOperatorAndDomain() throws Exception {
-    setupOperatorAndDomain("release/2.1", "2.1");
-    upgradeOperator(false);
-    testCompletedSuccessfully = true;
-    LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - " + testMethod);
-  }
-
-  /**
-   * Test for upgrading Operator from release 2.2.0 to develop branch.
-   *
-   * @throws Exception when upgrade fails
-   */
-  @Test
-  public void testOperatorUpgradeFrom2_2_0() throws Exception {
-    Assumptions.assumeTrue(FULLTEST);
-    testCompletedSuccessfully = false;
-    String testMethod = new Object() {
-    }.getClass().getEnclosingMethod().getName();
-    logTestBegin(testMethod);
-    OP_NS = "weblogic-operator220";
-    OP_DEP_NAME = "operator-upgrade220";
-    OP_SA = "operator-sa220";
-    DOM_NS = "weblogic-domain220";
-    DUID = "operatordomain220";
-    setupOperatorAndDomain("release/2.2", "2.2.0");
-    upgradeOperator(false);
-    testCompletedSuccessfully = true;
-    LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - " + testMethod);
-  }
-
-  /**
-   * Test for upgrading Operator from release 2.2.1 to develop branch.
-   *
-   * @throws Exception when upgrade fails
-   */
-  @Test
-  public void testOperatorUpgradeFrom2_2_1() throws Exception {
-    Assumptions.assumeTrue(FULLTEST);
-    testCompletedSuccessfully = false;
-    String testMethod = new Object() {
-    }.getClass().getEnclosingMethod().getName();
-    logTestBegin(testMethod);
-    OP_NS = "weblogic-operator221";
-    OP_DEP_NAME = "operator-upgrade221";
-    OP_SA = "operator-sa221";
-    DOM_NS = "weblogic-domain221";
-    DUID = "operatordomain221";
-    setupOperatorAndDomain("release/2.2.1", "2.2.1");
-    upgradeOperator(false);
-    testCompletedSuccessfully = true;
-    LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - " + testMethod);
-  }
-
-  /**
-   * Test for upgrading Operator from release 2.3.0 to develop branch.
+   * Test for upgrading Operator from release 2.5.0 to develop branch.
    *
    * @throws Exception when upgrade fails
    */
   @Test
-  public void testOperatorUpgradeFrom2_3_0() throws Exception {
+  public void testOperatorUpgradeFrom2_5_0() throws Exception {
     Assumptions.assumeTrue(QUICKTEST);
     testCompletedSuccessfully = false;
     String testMethod = new Object() {
     }.getClass().getEnclosingMethod().getName();
     logTestBegin(testMethod);
-    OP_NS = "weblogic-operator230";
-    OP_DEP_NAME = "operator-upgrade230";
-    OP_SA = "operator-sa230";
-    DOM_NS = "weblogic-domain230";
-    DUID = "operatordomain230";
-    setupOperatorAndDomain("release/2.3.0", "2.3.0");
-    upgradeOperator(false);
+    OP_NS = "weblogic-operator250";
+    DOM_NS = "weblogic-domain250";
+    namespaceList.append(OP_NS);
+    namespaceList.append(" ").append(DOM_NS);
+    OP_DEP_NAME = "operator-upgrade250";
+    OP_SA = "operator-sa250";
+    DUID = "operatordomain250";
+    setupOperatorAndDomain("release/2.5.0", "2.5.0");
+
+    M1_CTS = TestUtils.getCreationTimeStamp(DOM_NS,DUID + "-managed-server1");
```

Comment:
its kinda weird to get these here, and then print them in the upgradeOperator() method

Reply:
Move the CreateTimeStamp Info into @Test Method